app/controllers/tasks_controller.rb

```diff
@@ -2,7 +2,7 @@ class TasksController < ApplicationController
   get '/tasks' do
     tasks = current_user.tasks.all
     if is_json_request?
-      json tasks
+      json tasks.map { |task| {id: task.id, description: task.description, complete: task.complete} }
```

Comment:
If you define `as_json` in the `User` model, you won't need this in the controller.